src/guide/reactivity-computed-watchers.md

```diff
@@ -205,6 +205,24 @@ watch([fooRef, barRef], ([foo, bar], [prevFoo, prevBar]) => {
 })
 ```
 
+### Watching Reactive Objects
+
+Using a watcher to compare values of an Array or Object that are reactive requires that it has a copy made of just the values.
```

Comment:
**question**: would `deep` watcher option work with an Object in this case?

Reply:
so I hadn't specifically been testing `deep`, and I just made an experiment where I test deep vs not deep

https://codepen.io/fimion/pen/XWjdYpB

the behavior between the 2 are the same. I don't know if this is a bug, or if I am not nesting things correctly to trigger a need for `deep`.

Reply:
I commented and then I tested it with the function and now I see the difference. Let me rethink some of this and make sure I've got the edges covered.

Reply:
So, I've tried to add some clarification to this about `deep` and some of the weirdness of proxied objects. Let me know if I need to clarify it more?